packages/edit-post/src/editor.js

```diff
@@ -87,12 +89,13 @@ function Editor( {
 				focusMode: isFeatureActive( 'focusMode' ),
 				hasReducedUI: isFeatureActive( 'reducedUI' ),
 				hasThemeStyles: isFeatureActive( 'themeStyles' ),
-				preferredStyleVariations: getPreference(
+				preferredStyleVariations: getEditorPreference(
 					'preferredStyleVariations'
 				),
 				hiddenBlockTypes: getHiddenBlockTypes(),
 				blockTypes: getBlockTypes(),
 				__experimentalLocalAutosaveInterval: getPreference(
```

Comment:
Why aren't we using `getEditorPreference` here?

Reply:
I guess our medium term plan is to deprecate `getPreference` in `@wordpress/edit-post`, so, maybe the actual question is: why are we using `getEditorPreference` above? 😀

Reply:
I'm migrating `preferredStyleVariations` in a separate branch. For now they're still stored in the edit-post preferences. They're a bit more complicated, so definitely need to be done in an individual PR. 😅 

When that (and `panels`) are done I'll be able to drop `getEditorPreference` and it'll become a lot cleaner. 👍

Reply:
Ah right of course 👍